core/consensus/common/primitives/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "substrate-consensus-common-primitives"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "Common consensus primitives"
+edition = "2018"
+
+[dependencies]
+parity-codec = { version = "4.1.1", default-features = false }
+client = { package = "substrate-client", path = "../../../client", default-features = false }
+sr-primitives = { package = "sr-primitives", path = "../../../sr-primitives", default-features = false }
```

Comment:
```suggestion
sr-primitives = { path = "../../../sr-primitives", default-features = false }
```

Reply:
@bkchr I just merged this, sorry!  This is a good reminder to not rush in the future, though.

Reply:
I can do a last round of checks for the pattern:

`sr-primitives = { package = "sr-primitives", ...` in #3214

Reply:
too late xD

Reply:
Fixed within https://github.com/paritytech/substrate/pull/3236